nemo/collections/tts/models/fastspeech2.py

```diff
@@ -75,7 +75,12 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
 
         # Parser and mappings are used for inference only.
         self.parser = parsers.make_parser(name='en')
-        with open(cfg.mappings_filepath, 'r') as f:
+        if 'mappings_filepath' in cfg:
+            mappings_filepath = cfg.get('mappings_filepath')
+        else:
+            mappings_filepath = 'mappings.json'
+        mappings_filepath = self.register_artifact('mappings_filepath', mappings_filepath)
```

Comment:
This line will fail if `mapping.json` doesnt exist in the current working directory.

Reply:
I believe this should never happen, as the filepath should always be passed in by the config at training, and always restored from the .nemo when performing inference.

Reply:
Will add an error case to the else for clarity to the user